src/index.js

```diff
@@ -5,7 +5,11 @@
 import React from 'react'
 import {css, styleSheet} from 'glamor'
 import shouldForwardProperty from './should-forward-property'
-import domElements from './dom-elements'
+
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
```

Comment:
Yup, I'd prefer import too. But it doesn't work when I do `import htmlTagNames from 'html-tag-names'`. Is it because the modules (both `html-tag-names` & `svg-tag-names` doesn't have a default export? In package.json, `main` is pointing to a JSON file which has the array of tag names.